wordpress-mobile/WordPress-iOS
# Atomic Private Sites: add support for downloading the site icon.

This PR adds atomic-private support to site icons.

This is the second part of a larger change (#13770).

## To test:

1. Create an Atomic Private Site following the instructions from p58i-8xH-p2
2. Remember to make sure the site is indeed, well, private 😄 
3. Set a site icon for the site by going into WP-Admin > Appearance > Customize > Site Identity
4. Make sure the site icon is shown in WPiOS.

## PR submission checklist:
